pifi/videoprocessor.py

```diff
@@ -435,12 +440,26 @@ def __make_ffmpeg_to_python_fifo(self):
                 self.__FFMPEG_TO_PYTHON_FIFO_PREFIX + 'XXXXXXXXXX'
             )
         )
-        self.__logger.info('Making ffmpeg_to_python_fifo...')
+        self.__logger.info('Making ffmpeg_to_python_fifo {}...'.format(make_fifo_cmd))
         ffmpeg_to_python_fifo_name = (subprocess
             .check_output(make_fifo_cmd, shell = True, executable = '/usr/bin/bash')
             .decode("utf-8"))
         return ffmpeg_to_python_fifo_name
 
+    def __make_fps_fifo(self):
```

Comment:
could we have a function `__make_fifo(self, prefix)` to extract common logic with `__make_ffmpeg_to_python_fifo`?